contrib/install/local.cfg

```diff
@@ -0,0 +1,9 @@
+PACKAGES="load:dealii-prepare once:cmake once:astyle once:hdf5 once:p4est once:trilinos dealii"
```

Comment:
In the nonlinear solvers discussion, I will talk about SUNDIALS. One of the questions we'll have to discuss is whether we can add that to the prereqs -- my suggestion will be "yes", so perhaps add it here already now?

Reply:
@bangerth Why rely on SUNDIALS? NOX is part of Trilinos and of course already available without additional packages.

Reply:
We didn't recommend people to use NOX in the past (I'll have to update the deal.II docs accordingly -- patch coming up). So if you followed our recommendations, it wouldn't have included NOX.

But (i) Trilinos also doesn't include time steppers, (ii) my sense is that KINSOL is actually better than NOX, and (iii) NOX is not actively developed, whereas SUNDIALS is.